lib/provider.js

```diff
@@ -31,27 +49,30 @@ module.exports = {
       return this.scanFeaturesDir(results)
     }
   },
-  scanStepDefinitionsDir: function(results) {
+  scanStepDefinitionsDir: function(results = []) {
     //TODO: first search step definitions for your file
-    for (var step_def_file of fs.readdirSync(`${this.rootDirectory()}${this.featuresDirectory()}/step_definitions`)) {
+    const searchPath = `${this.rootDirectory()}${this.featuresDirectory()}/step_definitions`;
+    const allStepPaths = this.searchForPattern(searchPath, /.*/);
+    allStepPaths.forEach( step_def_file => {
       let data = fs.readFileSync(`${this.rootDirectory()}${this.featuresDirectory()}/step_definitions/${step_def_file}`, 'utf8');
```

Comment:
No problem! There is no rush for this change. Sure here is an example ruby steps file:
`features/step_definitions/role_steps.rb`
```
Given(/^I am a user with "(.*?)" and "(.*?)" roles$/) do |arg1, arg2|
  role_1 = Role.create(value: arg1)
  role_2 = Role.create(value: arg2)
end

Then(/^I should be able to access the admin page$/) do
  click_on 'Admin'
end

When(/^I am on the admin page$/) do
  page.must_have_content 'Admin'
end
```